File: 7KYU/Stanton measure/solution.cs
using System.Linq;
public class Kata
{
  /// <summary>
  /// Returns the Stanton Measure of the array</summary>
  /// <param name="arr">Int32[] to be evaluated.</param>
  public static int StantonMeasure(int[] arr)
  {
     var ones = arr.Where(x => x == 1).Count();
     return arr.Where(x => x == ones).Count();
  }
}

File: 7KYU/Stanton measure/solution.go
package kata

func StantonMeasure(arr []int) int {

	oneCounts := 0
	for i := 0; i < len(arr); i++ {
		if arr[i] == 1 {
			oneCounts = oneCounts + 1
		}
	}
	x := 0

	for i := 0; i < len(arr); i++ {
		if arr[i] == oneCounts {
			x = x + 1
		}
	}

	return x
}
